### PR TITLE
[INJIMOB-2538] fetch the openid4vp sharing flow client validation property from inji default properties file and pass it to library

### PR DESCRIPTION
**Pull request by PuBHARGAVI**
… 

## Description

> Wallet will fetch the openid4vp client validation property and pass it to the libraries and based on this value libraries will decide whether it should perform the client validation or not.

## Issue ticket number and link

> Example : [INJIMOB-2538](https://mosip.atlassian.net/browse/INJIMOB-2538)

[INJIMOB-2538]: https://mosip.atlassian.net/browse/INJIMOB-2538?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -34,13 +34,13 @@
...
         "branch" : "release-0.1.x",
-        "revision" : "9e7677440e561924289e4041522713d9dfe9dbd7"
+        "revision" : "2f12d1c12b87caa4aeb6894efd66d33893f1ebd8"
       }
     },
     {
       "identity" : "inji-vci-client-ios-swift",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/mosip/inji-vci-client-ios-swift",
+      "location" : "https://github.com/mosip/inji-vci-client-ios-swift/",
```
Comment (swatigoel) [inline ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved:43]: why added `/`

File: ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -49,7 +49,7 @@
     {
       "identity" : "pixelpass-ios-swift",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/mosip/pixelpass-ios-swift",
+      "location" : "https://github.com/mosip/pixelpass-ios-swift/",
```
Comment (swatigoel) [inline ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved:52]: why added `/`

File: ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 3]
```diff
@@ -70,7 +70,7 @@
       "location" : "https://github.com/valpackett/SwiftCBOR",
       "state" : {
         "branch" : "master",
-        "revision" : "ec24382864e5ffc6d3915c0818745d5ab12545a8"
+        "revision" : "6b8232d6437e3fe7df8cc46e5bb872f7066e7d0d"
```
Comment (swatigoel) [inline ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved:73]: why changed this?

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 1 above)
Comment (abhip2565) [reply ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved:43]: This file gets auto generated by xcode..No manual addition has been done

File: ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 3 above)
Comment (abhip2565) [reply ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved:73]: This is auto resolution of version. We have not specified any fixed commit.. so it resolved to the latest

**Review by swatigoel — APPROVED** [review]

**Pull request merged**